src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -292,6 +292,7 @@ public void addDebtToPerson(ReadOnlyPerson target, Debt amount) throws PersonNot
         try {
             Debt newDebt = new Debt(target.getDebt().toNumber() + amount.toNumber());
             editedPerson.setDebt(newDebt);
+            editedPerson.setTotalDebt(newDebt);
```

Comment:
It seems to me that debt and total debt are both set to newDebt, which is dependent on target.getDebt().
I think you should have newCurrentDebt and newTotalDebt since a person could payback some money before borrowing again, i.e. there is no guarantee that borrowing will cause current and total debt to be the same

Reply:
you are right